### DISCUSSION TITLE
Need a second or third option for how to show messages from teacher details

### DISCUSSION THREAD

**Discussion by tomfuller2** [Joomla-Bible-Study/Proclaim#923] [Ideas]
Right now only studieslist setup is an option. Need to allow a template file

**Comment by bcordis** [comment]
What are you looking at here?

**Comment by tomfuller2** [comment]
Adding to the dropdown any template files that have been created as an option. We should also have this from the series detail as well.

**Comment by bcordis** [comment]
Implemented in PR #1113 (merged to `development`).

The teacher detail page now has **three display modes** for messages, selectable via the `Show/Hide Teacher's Messages` template param:

| Value | Mode | Description |
|-------|------|-------------|
| 0 | Hide | No messages shown (default) |
| 2 | Grid Layout | Full configurable Cwmlisting grid (existing) |
| 3 | **Card Grid** | Bootstrap cards — thumbnail, title, date, scripture, series, intro excerpt. Responsive 1/2/3 columns. |
| 4 | **Simple List** | Compact rows — date, title, scripture. Column headers on desktop, stacked on mobile. |

Both new modes use the existing `teacherstudies` data — no new queries, no new params to configure. The `label_teacher` param (defaults to "Latest Messages") renders as a section heading for all modes.

Also fixed the default value (was `1`, which was never valid) and a "Messagess" typo in the language file.
